components.html

```diff
@@ -164,7 +164,10 @@ <h2> Students </h2>
                         <div class="row mb80 mb-xs-0">
                             <div class="col-md-8 col-md-offset-2 text-center">
                                 <h1>Connect with Fossasia on Social Media</h1>
+                                
```

Comment:
WTC? Also, remove this file from the commit.

Reply:
@liveHarshit The intend of Code In & especially for this site is to promote learning